src/Handler/Git.hs

```diff
@@ -234,8 +239,47 @@ gitPrintHandler (NHGit (GitRepoTrackable dir' _locSpec) nhg) = do
   liftIO $ putStrLn $ T.unpack $ format
 
 processGitNonInteractive :: GitRepoTrackable -> R.Reddup  ()
-processGitNonInteractive grTrack =
-  (lift $ checkGitProblems grTrack) >>= gitPrintHandler
+processGitNonInteractive grTrack = do
+  lift $ checkGitProblems grTrack >>= gitPrintHandler
+
+processGitNonInteractiveForce :: GitRepoTrackable -> R.Reddup ()
+processGitNonInteractiveForce grTrack =
+    lift . forever $
+        checkGitProblems grTrack >>= tryFixGitProblem
+
+tryFixGitProblem :: NHGit -> Tu.Shell ()
+tryFixGitProblem nh@(NHGit (GitRepoTrackable _dir _locSpec) nhg) = do
+    case nhg of
+        NHStatus (GP.Added _) ->
+            addAndWipCommit
+        NHStatus (GP.AddedAndModified _) ->
+            addAndWipCommit
+        NHStatus (GP.Staged _) ->
+            addAndWipCommit
+        NHStatus (GP.Unstaged _) ->
+            addAndWipCommit
+        NHStatus (GP.StagedAndUnstaged _) ->
+            addAndWipCommit
+        NHStatus (GP.Untracked _) ->
+            addAndWipCommit
+        NHStatus (GP.Deleted _) ->
+            addAndWipCommit
+        NHUnpushedBranch branch -> do
```

Comment:
I'm fine with all of these very similar things, but i wonder... it looks like they're all the same except for the `Unknown` clause (which would be handled in the fall-through). I would say its not super important and out of scope for this PR but i think really the `unknown` case should be a parse error and I should fail in the parsing. That would simplify this of course, but it would probably also simplify other things. if and when i get around to getting back to working on this some more I will look at that I think, the parsing is some of the oldest code.